batch/batch/driver/job.py

```diff
@@ -155,6 +162,30 @@ async def mark_job_started(app, batch_id, job_id, attempt_id, instance, start_ti
     await add_attempt_resources(db, batch_id, job_id, attempt_id, resources)
 
 
+async def mark_job_creating(app, batch_id, job_id, attempt_id, instance, start_time, resources):
+    db: Database = app['db']
+
+    id = (batch_id, job_id)
+
+    log.info(f'mark job {id} creating')
+
+    try:
+        rv = await db.execute_and_fetchone(
+            '''
+CALL mark_job_creating(%s, %s, %s, %s, %s);
+''',
+            (batch_id, job_id, attempt_id, instance.name, start_time))
+    except Exception:
+        log.exception(f'error while marking job {id} creating on {instance}')
+        raise
```

Comment:
This function, `mark_job_creating`, is called once. At that call-site, the exception is caught and logged as `info`. Is this exception actually exceptional?